### ISSUE TITLE
Formula of `-NamedRange` evaluates incorrectly in v6

### ISSUE THREAD

**Issue by craigbrown** [EPPlusSoftware/EPPlus#1187]
Hi EPPlus team! Hope you're good.

In EPPlus v6, if a formula starts with a negative/minus sign which is immediately followed by a named range reference, the formula is evaluated as if the minus sign wasn't there. Here's some examples:

```csharp
[TestMethod]
public void MinusNamedRange()
{
    using (var pck = new ExcelPackage())
    {
        var sheet1 = pck.Workbook.Worksheets.Add("Sheet1");
        sheet1.Cells["A1"].Value = 123456;

        sheet1.Names.Add("MyRange", sheet1.Cells["A1"]);

        sheet1.Cells["C3"].Formula = "-MyRange";

        pck.Workbook.Calculate();

        Assert.AreEqual(-123456, sheet1.Cells["C3"].GetValue<double>(), 1E-5); //ERROR: evaluates to 123456
    }
}
[TestMethod]
public void MinusNamedRangePlusNamedRange()
{
    using (var pck = new ExcelPackage())
    {
        var sheet1 = pck.Workbook.Worksheets.Add("Sheet1");
        sheet1.Cells["A1"].Value = 123456;
        sheet1.Cells["B1"].Value = 3;

        sheet1.Names.Add("MyRange", sheet1.Cells["A1"]);
        sheet1.Names.Add("Another", sheet1.Cells["B1"]);

        sheet1.Cells["C3"].Formula = "-MyRange+Another";

        pck.Workbook.Calculate();

        Assert.AreEqual(-123453, sheet1.Cells["C3"].GetValue<double>(), 1E-5); //ERROR: evaluates to 123459
    }
}
```

This issue is only present in v6; it seems to work fine in v7. Not sure if you're still supporting v6 but it looks like you're still releasing updates so I thought I would report it.

**Comment by JanKallman** [comment]
Thanks for reporting it.
We are still maintaining EPPlus 6, so we will have a look at it.

**Comment by swmal** [comment]
@craigbrown - I have added a fix for this, see PR #1189.

We can't fix this for multi cell ranges in v6, only for names that refers to a single cell. The reason for this is that v6 doesn't make a copy of the source range as v7 does. But the two tests you provided passes now. The fix will be included in next release of v6.

**Comment by craigbrown** [comment]
Thanks!
